.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
You will need to specify the version as we do not support anything less than python 3.7.

Reply:
```suggestion
        python-version: '3.x'
```

Reply:
```suggestion
        python-version: '3.7.'
```

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/7105/files/8758dc442a073b59417c1407c3d4db4c2fb4e1b7#r663290462

Reply:
3.0.7